src/components/ModularTable/ModularTable.tsx

```diff
@@ -35,7 +36,7 @@ function ModularTable({
   const showEmpty: boolean = emptyMsg && (!rows || rows.length === 0);
 
   return (
-    <Table {...getTableProps()}>
+    <Table className="p-modular-table" {...getTableProps()}>
```

Comment:
Will that classname prop get overridden by `getTableProps()`?

Reply:
As far as I can tell, `getTableProps()` only returns `{ role: "table" }`. Still, after looking at the docs I think the correct way of handling this is adding the additional prop directly into the `getTableProps()` function.